__main.cfg

```diff
@@ -1,5 +1,6 @@
 #textdomain wesnoth-millennium-era
 
+# This file is not used for ageless, so it cannot be named _main.cfg
```

Comment:
is it fine when I just delete this file?

Reply:
Yes, I do not use it.